src/typescript/06-types/types.md

```diff
@@ -601,26 +568,24 @@ In this exercise, we will create our own typed variable by updating `2b-date-exp
 
 - Use `new Date()` to create an instance of Date.
 
-### Verify Your Solution
+### Verify
 
-✏️ Run the following to verify your solution:
+Create **date-exports.test.ts** and update it to be:
+
+@sourceref ../../../exercises/typescript/06-types/02-problem/src/types/date-exports.test.ts
 
 ```shell
-npm run 2b-types
+npm run test
 ```
 
 ### The solution
 
 <details>
 <summary>Click to see the solution</summary>
 
-✏️ Update `2b-date-export.ts` to create, assign, and export a date variable.
+✏️ Update **date-export.ts** to create, assign, and export a date variable.
 
-```typescript
-let me: Date;
-me = new Date("6-11-1993");
-export default me;
-```
+@sourceref ../../../exercises/typescript/06-types/02-solution/src/types/date-exports.test.ts
```

Comment:
This doesn’t look right… the solution should be the code changes they make to the implementation, not the test.

Looking at this exercise a bit more, I think it needs some serious reworking. I would fix this one comment and leave the rest of it as-is. We can talk through it next week.